accepted/00083-containerized-proxy.md

```diff
@@ -0,0 +1,82 @@
+- Feature Name: Deliver Proxy as a container
+- Start Date: 2021-10-11
+
+# Summary
+[summary]: #summary
+
+Deliver Proxy as a container application.
+
+# Motivation
+[motivation]: #motivation
+
+There are deployment scenarios in which adding a new Virtual Machine to implement Proxy functionality is cumbersome or impossible, while leveraging an existing container runtime cold be an easier option, especially in highly distributed ("Edge") environments.
+
+Having the option of deploying Proxy functionality as a container can be easier to operate (eg. deployment, independence from underlying OS, keeping up to date) and more flexible (eg. implementing active/passive HA by leveraging an existing Kubernetes environment).
+
+It is believed implementing Proxy containerization will surface a subset of common problems and solutions also useful to implement a containerized Server in future.
+
+# Detailed design
+[design]: #detailed-design
+
+## Delivery
+  - Proxy is delivered as one container
+    - Updates are delivered via new images on a public Registry
+  - Documentation is produced to describe how this container can be run on popular conainer runtimes: [docker](https://docs.docker.com/engine/reference/commandline/cli/) and [podman](https://podman.io/)
+  - A minimal [Helm](https://helm.sh/) chart is delivered and tested against [k3s](https://k3s.io/)
+
+## Onboarding
+  - Containerized Proxy is not a managed client, does not run `salt-minion`, is not registered in the usual way
+  - A new UI page (and API endpoint) allows to add containerized Proxies by specifying their FQDN
+    - After confirmation a System entry (in the System list) is created 
```

Comment:
Resolving per chat

Reply:
From a UX perspective I can agree with that. Problem is that all existing Proxy functionality is bolted on the assumption there is a system profile (the Proxy page, Proxy related DB tables and so on).

It is not clear to me at this point how much bigger of a change it would be not to have a system profile. We should take a look and at the very least mention your approach as an alternative in this RFC - or adopting it.